python/tests/test_highlevel.py

```diff
@@ -743,6 +746,16 @@ def test_compute_mutation_parent(self):
             parent = ts.tables.mutations.parent
             self.assertTrue(np.array_equal(parent, before))
 
+    def test_compute_mutation_time(self):
+        for ts in get_example_tree_sequences():
```

Comment:
This will fail, I think, when msprime generates real times.

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/93e65c6ddc6f4768c1ac07576e62979bb5a04604